docker/standalone/Dockerfile

```diff
@@ -8,6 +8,7 @@ FROM amazoncorretto:17
 LABEL maintainer="engineering@littlehorse.io"
 
 ENV PATH ${PATH}:/kafka/bin:/lh
+VOLUME /data
```

Comment:
What happens if you want the data to be stored locally inside the container, not on a mounted volume? Does this still work?

Reply:
if the stop de container (manually or because you restarted your compu) and then you run docker start the data will be there, they can run docker rm <dockername litthlehorse in the example> and then run again it will create a new volume

Reply:
testing without this line again

Reply:
you are right, not needed, much better without it, removing it